docs/03.API-EXAMPLE.md

```diff
@@ -214,6 +214,7 @@ print_value (const jerry_value_t value)
     jerry_char_t str_buf_p[req_sz];
 
     jerry_string_to_char_buffer (value, str_buf_p, req_sz);
+    str_buf_p[req_sz] = '\0';
```

Comment:
It's possible, but this declaration method is also used in [main-unix.c:453](https://github.com/Samsung/jerryscript/blob/master/jerry-main/main-unix.c#L453)